src/maintenance/azext_maintenance/generated/commands.py

```diff
@@ -0,0 +1,74 @@
+        'rations.{}',
+        client_factory=cf_apply_update)
+    with self.command_group('maintenance apply-update', maintenance_apply_update, client_factory=cf_apply_update,
+                            is_experimental=True) as g:
+        g.custom_show_command('show', 'maintenance_apply_update_show')
+        g.custom_command('create', 'maintenance_apply_update_create')
+        g.custom_command('update', 'maintenance_apply_update_update')
+        g.custom_command('get-parent', 'maintenance_apply_update_get_parent')
+
+    from azext_maintenance.generated._client_factory import cf_configuration_assignment
+    maintenance_configuration_assignment = CliCommandType(
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._configuration_assignment_operations#Co'
+        'nfigurationAssignmentOperations.{}',
+        client_factory=cf_configuration_assignment)
+    with self.command_group('maintenance configuration-assignment', maintenance_configuration_assignment,
+                            client_factory=cf_configuration_assignment, is_experimental=True) as g:
+        g.custom_command('list', 'maintenance_configuration_assignment_list')
+        g.custom_command('create', 'maintenance_configuration_assignment_create')
+        g.custom_command('update', 'maintenance_configuration_assignment_update')
+        g.custom_command('delete', 'maintenance_configuration_assignment_delete', confirmation=True)
+        g.custom_command('list-parent', 'maintenance_configuration_assignment_list_parent')
+
+    from azext_maintenance.generated._client_factory import cf_maintenance_configuration
+    maintenance_maintenance_configuration = CliCommandType(
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._maintenance_configuration_operations#M'
+        'aintenanceConfigurationOperations.{}',
+        client_factory=cf_maintenance_configuration)
+    with self.command_group('maintenance maintenance-configuration', maintenance_maintenance_configuration,
+                            client_factory=cf_maintenance_configuration, is_experimental=True) as g:
+        g.custom_command('list', 'maintenance_maintenance_configuration_list')
+        g.custom_show_command('show', 'maintenance_maintenance_configuration_show')
+        g.custom_command('create', 'maintenance_maintenance_configuration_create')
+        g.custom_command('update', 'maintenance_maintenance_configuration_update')
+        g.custom_command('delete', 'maintenance_maintenance_configuration_delete', confirmation=True)
+
+    from azext_maintenance.generated._client_factory import cf_update
+    maintenance_update = CliCommandType(
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._update_operations#UpdateOperations.{}',
+        client_factory=cf_update)
+    with self.command_group('maintenance update', maintenance_update, client_factory=cf_update,
+                            is_experimental=True) as g:
```

Comment:
Don't label the existing commands as experimental as they are GA before; other existing commands as well.

Reply:
Please just update is_experimental label in "Generated" folder as there is no good mechanism to overwrite it.

---
In reply to: [478229448](https://github.com/Azure/azure-cli-extensions/pull/2214#discussion_r478229448) [](ancestors = 478229448)